graphein/protein/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .config import *
+from .edges import *
+from .features import *
+from .graphs import *
+from .resi_atoms import *
+from .utils import *
```

Comment:
The `*` imports might not be the most ideal. I think the `__init__.py` is a great opportunity to define what exactly should be made high-level for a user. For example, in `protein`, I think `construct_graph` is a good one for the high-level API, as well as the relevant Config object. What do you think?

Reply:
I think you're right on this. Will keep this for now - I'm rather keen to consolidate some of the outstanding PRs. You've raised an issue on specifying what goes into each namespace more carefully. Perhaps this is best done after we've established the remaining functionality?

Reply:
Yep, that sounds about right to me.